orca-sql/src/main/resources/db/changelog/20240731-add-compression-type-column-mariadb.yaml

```diff
@@ -0,0 +1,29 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-pipelines-compressed-executions-mariadb
+      author: juangod-wise
+      changes:
```

Comment:
Do we need something like:
```
      preConditions:
        onFail: MARK_RAN
        not:
          columnExists:
            tableName: pipelines_compressed_executions
            columnName: compression_type
```
like the [postgres change](https://github.com/spinnaker/orca/blob/eb61f0e25330b2c2a34d543e63e59f6be95de859/orca-sql/src/main/resources/db/changelog/20220427-add-compression-type-column-to-compressed-executions-tables-postgres.yaml#L13-L18) has?

Reply:
I don't believe this is necessary because in MariaDB with the current given migrations no column would have been created. The migration won't be ran twice because of the changeset checksums. In Wise we have this migration in Prod for a couple of months now and we had no issue. The only scenario I can think of where it may be useful is if someone pulls this migration in, has MariaDB, and already has another migration that created the same column. But it would have been amended when the OSS changes are merged in.. I could add it if you want, to be extra careful. As you wish :) .

Reply:
OK, I think I see it.  The [postgres changeset](https://github.com/spinnaker/orca/blob/eb61f0e25330b2c2a34d543e63e59f6be95de859/orca-sql/src/main/resources/db/changelog/20220427-add-compression-type-column-to-compressed-executions-tables-postgres.yaml#L10) isn't actually specific to postgres. It doesn't have something like
```
      preConditions:
        onFail: CONTINUE
        dbms:
          type: postgres
```
I think we need something like that here.  There could easily be a better way to express it.

Reply:
Yes please.  The important thing is to add a precondition that this changeset only runs for mariadb, and then if the precondition fails, to continue.

Reply:
But by having `dbms: mariadb` on all changesets, it already only runs for mariadb. How do we know this? Because this bug happened because `20201214-create-compressed-executions-table.yaml` did not run the changesets that contained 

```
    - sql:
          dbms: mysql
```

Reply:
fair enough.